src/Sarif/HashUtilities.cs

```diff
@@ -34,7 +34,12 @@ public static void ComputeHashes(string fileName, out string md5, out string sha
 
                         var sha256Cng = new SHA256Cng();
                         checksum = sha256Cng.ComputeHash(bufferedStream);
-                        sha256 = BitConverter.ToString(checksum).Replace("-", String.Empty);                            
+                        sha256 = BitConverter.ToString(checksum).Replace("-", String.Empty);
+
+                        // disposals
+                        md5Cng.Dispose();
+                        sha1Cng.Dispose();
+                        sha256Cng.Dispose();                        
```

Comment:
As we've discussed: Use `using` statements to ensure that `Dispose` is called even if the intervening code throws an exception.